service/discordBaseService_test.go

```
@@ -0,0 +1,65 @@
+		DiscordBaseService(w, r)
+		assert.Equal(t, http.StatusOK, w.Code)
+	})
+
+	t.Run("should return success response when message type is interaction ping", func(t *testing.T) {
+		msgByte, err := json.Marshal(dtos.DiscordMessage{Type: discordgo.InteractionPing})
+		assert.NoError(t, err)
+		r := httptest.NewRequest("POST", "/", bytes.NewBuffer([]byte(msgByte)))
+		w := httptest.NewRecorder()
+		DiscordBaseService(w, r)
+		assert.Equal(t, http.StatusOK, w.Code)
+		assert.Equal(t, fmt.Sprintln(fmt.Sprintf(`{"type":%d}`, discordgo.InteractionResponsePong)), w.Body.String())
```

golangci-lint (1.64.8)

S1038: should use fmt.Sprintf instead of fmt.Sprintln(fmt.Sprintf(...)) (but don't forget the newline)

(gosimple)